aries-test-harness/features/0037-present-proof.feature

```diff
@@ -0,0 +1,94 @@
+Feature: Aries agent present proof functions RFC 0037
+
+   @T001-API10-RFC0037 @P1 @AcceptanceTest @wip @NeedsReview
+   Scenario Outline: Present Proof where the prover does not propose a presentation of the proof and is acknowledged
+      Given "2" agents
+         | name  | role     |
+         | Faber | verifier |
+         | Bob   | prover   |
+      And "Faber" and "Bob" have an existing connection
+      And "Bob" has an issued credential from <issuer>
+      When "Faber" sends a request for proof presentation to "Bob"
+      And "Bob" makes the presentation of the proof
+      And "Faber" acknowledges the proof
+      Then "Bob" has the proof acknowledged
+
+      Examples:
+         | issuer |
+         | Acme   |
+         | Faber  |
+
+   @T001.2-API10-RFC0037 @P1 @AcceptanceTest @wip @NeedsReview
+   Scenario Outline: Present Proof of specific types and proof is acknowledged
+      Given "2" agents
+         | name  | role     |
+         | Faber | verifier |
+         | Bob   | prover   |
+      And "Faber" and "Bob" have an existing connection
+      And "Bob" has an issued credential from <issuer>
+      When "Faber" sends a request for proof presentation to "Bob" for <proof>
```

Comment:
@swcurran Does this line make sense given what we talked about in the standup this morning? <proof> represents the types in the examples below, each one will probably have its own cred schema or maybe one schema with all?